techmatters/hrm
# Add taxonomies to high boost fields

## Description
This PR adds taxanomies to high boost fields.

### Related Issues
Fixes https://tech-matters.atlassian.net/browse/CHI-2212

### Verification steps
Check that Resource Search in `aselo development` behaves slightly differently than `KHP staging`
